UPGRADE-3.x.md

```diff
@@ -1,6 +1,11 @@
 UPGRADE 3.x
 ===========
 
+## Deprecated SonataAdminExtension::output()
+
+The `SonataAdminExtension::output()` method is deprecated and should not be
```

Comment:
if there is a replacement for output, you should also include that. Maybe: and should not be used anymore, instead use `SonataAdminExtension::render()`

Reply:
Sorry, I didn't saw that the render method is private. Don't mind my comment

Reply:
Too late... :-P

Reply:
But the render method is private

Reply:
@TimoBakx Can you remove it from the upgrade note, we didn't see it was private (it can't replace the old public method), sorry for the misunderstanding.

Reply:
No worries. I updated the note.